packages/www/pages/dashboard/developers/webhooks/[id].tsx

```diff
@@ -121,7 +121,7 @@ const ApiKeys = () => {
                             setDeleting(true);
                             await deleteWebhook(data.id);
                             setDeleting(false);
-                            revalidate();
+                            queryClient.invalidateQueries();
```

Comment:
we don't want to invalidate ALL queries across the app. We should pass the queryKey to `invalidateQueries`

Reply:
Also, we should `await` this, if `invalidateQueries` returns a promise.